Chapter1/README.md

```diff
@@ -0,0 +1,233 @@
+1. If your account is present in more than one Azure AD B2C tenant, select your profile at the top right corner in the menu on top of the page, and then **switch directory** to change your portal session to the desired Azure AD B2C tenant.
+
+### Register the app
+
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD B2C** service.
+1. Select the **App Registrations** blade on the left, then select **New registration**.
+1. In the **Register an application page** that appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `b2c-management-spa`.
+   - Under **Supported account types**, select **Accounts in this organizational directory only**.
+   - In the **Redirect URI (optional)** section, select **Single-page application** in the combo-box and enter the following redirect URI: `http://localhost:3000`.
+1. Select **Register** to create the application.
+1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. Select **Save** to save your changes.
+1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs.
+   - Select the **Add a permission** button and then,
+   - Ensure that the **Microsoft APIs** tab is selected.
+   - In the *Commonly used Microsoft APIs* section, select **Microsoft Graph**
+   - In the **Delegated permissions** section, select the **User.ReadWrite.All** in the list. Use the search box if necessary.
+   - Select the **Add permissions** button at the bottom.
+   - Finally, grant **Admin consent** to these permissions.
+
+#### Configure the code to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `App\authConfig.js` file.
+1. Find the key `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `b2c-management-spa` app copied from the Azure portal.
+1. Find the key `Enter_the_Tenant_Info_Here` and replace the existing value with your tenant ID copied form Azure portal.
+
+> :information_source: If you are using an existing B2C app registration that you use for signing-in users with user-flows, replace `Enter_the_Tenant_Info_Here` not with your tenant ID, but with "common".
+
+## Running the sample
+
+Locate the root folder of the sample in a terminal. Then:
+
+```console
+    cd Chapter1
+    npm start
+```
+
+## Explore the sample
+
+1. Open your browser and navigate to `http://localhost:3000`.
+1. Click on the **sign-in** button on the top right corner.
+1. Click on the **Get Users** button retrieve the users in your tenant.
```

Comment:
>users in your tenant. [](start = 50, length = 21)

also talk about the "Add user" experience, with a note on using verified domain names